static.config.js

```diff
@@ -353,26 +94,29 @@ export default {
       },
       {
         path: '/reference',
-        component: 'src/pages/reference',
+        component: 'src/pages/referenceRedirect',
         getData: async () => ({
-          title: '@neo-one/client',
-          type: 'All',
-          content: {
-            type: 'referenceItems',
-            value: refChildren,
-          },
-          current: '@neo-one/client',
-          sidebar: refSidebar,
+          redirect: references[0].current,
         }),
-        children: refChildren.map((ref) => ({
-          path: ref.slug.slice('/reference/'.length),
+        children: references.map((moduleRef) => ({
+          path: moduleRef.slug.slice('/reference/'.length),
           component: 'src/pages/reference',
           getData: async () => ({
-            title: ref.name,
-            content: { type: 'referenceItem', value: ref },
-            current: `@neo-one/client`,
-            sidebar: refSidebar,
+            title: moduleRef.title,
+            content: moduleRef.content,
+            current: moduleRef.current,
+            sidebar: moduleRef.sidebar,
           }),
+          children: moduleRef.content.value.map((ref) => ({
```

Comment:
Don't do any additional processing in this file - just return the data in the expected format directly from `getReferences` (see `getBlogs` for an example)